components/organisms/ChangePasswordUser/ChangePasswordUser.tsx

```diff
@@ -78,27 +46,20 @@ export const ChangePasswordUser = () => {
     }
 
     try {
-      const responseUpdate = await fetch(
-        `${process.env.NEXT_PUBLIC_BACKEND_URL}/auth/change-password/${session?.user?.uid}`,
-        {
-          method: "PUT",
-          headers: {
-            "Content-Type": "application/json",
+      if (userInfo?.uid !== undefined) {
+        const changePasswordUser = await changePassword(
+          userInfo?.uid,
+          password
+        );
+        Swal.fire({
+          icon: "success",
+          title: "Contraseña modificada",
+          text: "Los cambios en tu perfil han sido guardados exitosamente",
+          didClose: () => {
```

Comment:
The name of this attribute isn't clear, can you change it for any other more descriptive? pls  

It looks like a function to redirect after closing the modal, so I suggest something like `redirectAfterClose`

Reply:
included attribute of swil.fire and next